modules/schema/src/main/java/org/apache/ignite/internal/schema/configuration/SchemaDescriptorConverter.java

```diff
@@ -151,11 +160,13 @@ private static Serializable convertDefault(NativeType type, String dflt) {
             case DOUBLE:
                 return Double.parseDouble(dflt);
             case DECIMAL:
-                return new BigDecimal(dflt);
+                return new BigDecimal(dflt).setScale(((DecimalNativeType)type).scale(), RoundingMode.HALF_UP);
```

Comment:
SQL standard does not specify whether number should be rounded or truncated.
But we must mention this in documentation and ColumnType javadoc.

Reply:
I've created a ticket IGNITE-15217 to improve this.